Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheet.swift

```diff
@@ -49,7 +49,7 @@ class PhotonActionSheet: UIViewController {
 
     // MARK: - Variables
     private var tableView = UITableView(frame: .zero, style: .grouped)
-    private var viewModel: PhotonActionSheetViewModel!
+    var viewModel: PhotonActionSheetViewModel!
```

Comment:
Does this need to have a `!`? We should avoid guarantees where possible, such that Xcode screams at us properly.

Reply:
It's a required init param, I'll change it to a let.